lib/game.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/ship'
+require './lib/cell'
+require './lib/board'
+
```

Comment:
Looks good, let's keep outlining the frame of the game methods on this file and delegate it to more classes when we have time.